analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SonarLintXmlReaderTest.cs

```diff
@@ -90,10 +101,25 @@ public void SonarLintXmlReader_PartiallyMissingProperties_ExpectedAndDefaultValu
     public void SonarLintXmlReader_CheckEmpty_DefaultBehaviour() =>
         CheckSonarLintXmlReaderDefaultValues(SonarLintXmlReader.Empty);
 
+    [TestMethod]
+    public void SonarLintXmlReader_LanguageDoesNotExist_Throws()
+    {
+        var sut = CreateSonarLintXmlReader($"ResourceTests\\SonarLintXml\\All_Properties_cs\\SonarLint.xml");
+        sut.Invoking(x => x.IgnoreHeaderComments("fsharp"))
```

Comment:
To keep it authentic and aligned with all other UTs.
Also below.
```suggestion
        sut.Invoking(x => x.IgnoreHeaderComments(LanguageNames.FSharp))
```